linear_api/managers/issue_manager.py

```
@@ -738,14 +742,125 @@ def get_comments(self, issue_id: str) -> List[Comment]:
+        }
+        for header in upload_info['headers']:
+            headers[header['key']] = header['value']
+
+        async with aiohttp.ClientSession() as session:
+            async with session.put(
+                upload_info['uploadUrl'], 
+                headers=headers, 
+                data=file_data
+            ) as response:
+                if response.status not in (200, 201):
+                    raise Exception(f"Upload failed with status {response.status}")
```

Ruff (0.14.10)

Create your own exception

(TRY002)